Contents/mods/Perdition/media/lua/server/Items/Perdition_distributions.lua

```diff
@@ -14,7 +14,46 @@ local function preDistributionMerge()
                 "Perdition.CleaverUpgradeManual", 1,
                 "Perdition.ShovelUpgradeManual", 1,
             }
+        },
+        counter = {
+            rolls = 4,
```

Comment:
This replaces the rolls value in the procedural distributions list.  More rolls = higher chances at obtaining rare items.